hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/TestHoodieSparkUtils.scala

```diff
@@ -103,4 +106,34 @@ class TestHoodieSparkUtils {
     assertEquals(files.sortWith(_.toString < _.toString), indexedFilePaths.sortWith(_.toString < _.toString))
     spark.stop()
   }
+
+  @Test
+  def testCreateRdd(@TempDir tempDir: File): Unit = {
```

Comment:
Can you rename this method to something that you're trying to test about createRDD (here schema evolution) ?